--- prettify.js	2018-04-30 03:23:30.000000000 +0800
+++ prettify.new.js	2018-12-28 15:15:20.000000000 +0800
@@ -166,7 +166,7 @@
 var prettyPrint;


-(function () {
+// (function () {
   var win = (typeof window !== 'undefined') ? window : {};
   // Keyword lists for various languages.
   // We use things that coerce to strings to make them compact when minified
@@ -1702,7 +1702,7 @@
    * Contains functions for creating and registering new language handlers.
    * @type {Object}
    */
-  var PR = win['PR'] = {
+  /*var*/ PR = win['PR'] = {
         'createSimpleLexer': createSimpleLexer,
         'registerLangHandler': registerLangHandler,
         'sourceDecorator': sourceDecorator,
@@ -1741,10 +1741,10 @@
   // whose value is an object. This helps avoid conflict with any
   // other existing JavaScript code that could have defined a define()
   // function that does not conform to the AMD API.
-  var define = win['define'];
-  if (typeof define === "function" && define['amd']) {
-    define("google-code-prettify", [], function () {
-      return PR;
-    });
-  }
-})();
+//   var define = win['define'];
+//   if (typeof define === "function" && define['amd']) {
+//     define("google-code-prettify", [], function () {
+//       return PR;
+//     });
+//   }
+// })();
